contrib/ci/local.cfg

```diff
@@ -0,0 +1,9 @@
+# Compile with debug checks, but also with optimizations to make tests run fast
+NATIVE_OPTIMIZATIONS=ON
+DEAL_II_CONFOPTS="-DDEAL_II_WITH_COMPLEX_VALUES=OFF -DCMAKE_BUILD_TYPE='Debug' -DDEAL_II_CXX_FLAGS_DEBUG='-ggdb -Wa,--compress-debug-sections -O3' -DDEAL_II_COMPONENT_EXAMPLES=OFF"
```

Comment:
Is it enough to only have the debug version of deal.II? I guess we do not compile ASPECT in release mode in any of the tests, right?